### PR TITLE
Creates donation widget

### PR DESCRIPTION
**Pull request by adamclark-dev**
Type: patch

https://deploy-preview-249--cr-component-library.netlify.com/#!/Donate/1

## Changes proposed in this PR

- Create new donation widget with giving selector for A/B testing
- Creates Organsims Foledring and moves modules into Organsims

### CODE REVIEW

File: src/components/Molecules/HeaderEsuWithIcon/HeaderEsuWithIcon.style.js [hunk 1]
```diff
@@ -1,7 +1,7 @@
 import styled from 'styled-components';
 import Icon from '../../Atoms/SocialIcons/Icon/Icon';
 import spacing from '../../../theme/shared/spacing';
-import EmailSignUp from '../EmailSignUp/EmailSignUp';
+import EmailSignUp from '../../Organisms/EmailSignUp/EmailSignUp';
```
Comment (AndyEPhipps) [inline src/components/Molecules/HeaderEsuWithIcon/HeaderEsuWithIcon.style.js]: Is this not just a Component, to keep it inline with existing stuff? Or should we revisit naming?

Does it matter?

Does anything?

_Ad Astra_ messed me up, man.

Comment (AndyEPhipps) [reply src/components/Molecules/HeaderEsuWithIcon/HeaderEsuWithIcon.style.js]: Ah, you already renamed it all already.

Tommy Lee Jones would be proud.

**Comment by gusliedke** [comment]
Waiting for Design changes

**Pull request merged**